UnitySetup/UnitySetup.psd1

```diff
@@ -1,158 +1,161 @@
-            # IconUri = ''
-
-            # ReleaseNotes of this module
-            # ReleaseNotes = ''
-
-            # Prerelease string of this module
-            # Prerelease = ''
-
-            # Flag to indicate whether the module requires explicit user acceptance for install/update
-            # RequireLicenseAcceptance = $false
-
-            # External dependent modules of this module
-            # ExternalModuleDependencies = @()
-
-        } # End of PSData hashtable
-    
-    } # End of PrivateData hashtable
-
-    # HelpInfo URI of this module
-    HelpInfoURI       = 'https://github.com/Microsoft/unitysetup.powershell/blob/master/README.md'
-
-    # Default prefix for commands exported from this module. Override the default prefix using Import-Module -Prefix.
-    # DefaultCommandPrefix = ''
-
-}
-
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+# Module manifest for module 'PSGet_UnitySetup'
+#
+# Generated by: Josh Wittner
+#
+# Generated on: 2018-01-31
+#
+
+@{
+
+    # Script module or binary module file associated with this manifest.
+    RootModule        = 'UnitySetup'
+
+    # Version number of this module.
+    ModuleVersion     = '5.6'
+
+    # Supported PSEditions
+    # CompatiblePSEditions = @()
+
+    # ID used to uniquely identify this module
+    GUID              = '6dc524ef-5f56-4bc3-b04d-3c2906898f40'
+
+    # Author of this module
+    Author            = 'Josh Wittner'
+
+    # Company or vendor of this module
+    CompanyName       = 'Microsoft'
+
+    # Copyright statement for this module
+    Copyright         = 'Copyright (c) Microsoft Corporation. All rights reserved.'
+
+    # Description of the functionality provided by this module
+    Description       = 'Tools for managing and automating your Unity installs and projects.'
+
+    # Minimum version of the Windows PowerShell engine required by this module
+    # PowerShellVersion = ''
+
+    # Name of the Windows PowerShell host required by this module
+    # PowerShellHostName = ''
+
+    # Minimum version of the Windows PowerShell host required by this module
+    # PowerShellHostVersion = ''
+
+    # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # DotNetFrameworkVersion = ''
+
+    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # CLRVersion = ''
+
+    # Processor architecture (None, X86, Amd64) required by this module
+    # ProcessorArchitecture = ''
+
+    # Modules that must be imported into the global environment prior to importing this module
+    RequiredModules   = @(
+        @{ModuleName = "powershell-yaml"; ModuleVersion = "0.3"; Guid = "6a75a662-7f53-425a-9777-ee61284407da" },
+        @{ModuleName = "Az.Accounts"; ModuleVersion = "2.15.1"; }
```

Comment:
Oh, lets put the guid in here too - for Az.Accounts its should be: `17a2feff-488b-47f9-8729-e2cec094624c` from https://www.powershellgallery.com/packages/Az.Accounts/2.15.1/Content/Az.Accounts.psd1.